tests/test__config.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
remove the second underscore from the file name. The name should only have one underscore i.e. tests/test_config.py